RELEASE_NOTES.md

```diff
@@ -19,7 +19,56 @@
 
 Version v0.2.0 -- Release date 2020-06-20
 ==========================================
-Lots of stuff to fill in...
+* Extensive updates to Data Repository.
+* Discretization:
+  * Finite Element Interface
+    * Kernel launching interface looping abstraction.
+    * Element Formulations for 1st order:
```

Comment:
This is totally up to you, but the capitalization style in these notes is all over the map.  Perhaps we can just capitalize the first word in each phrase unless it is a proper object like `Data Repository`.  Otherwise, I would just write `Finite volume stencils for classical FVM formulation`, `Element formulations for 1st order`, etc.  This obviously me being OCD.  We can leave as is if you prefer.

Reply:
I think I took care of this now.